### PR TITLE
docs: instructions on how to install on Windows + `conda`

### PR DESCRIPTION
**Pull request by MthwRobinson**
### Summary

Adds documentation on how to install `unstructured` on Windows with `conda`, along with optional dependencies.

### Testing

In a fresh virtual environment on a Windows machine, follow the instructions in the documentation to install `unstructured` and all of the optional dependencies.

### CODE REVIEW

File: docs/source/installing.rst [hunk 1]
```diff
@@ -6,13 +6,85 @@ root directory. Developers can run ``make install-local`` to install the dev and
 requirements alongside the base requirements. If you want a minimal installation without any
 parser specific dependencies, run ``make install-base``.
 
+
+Installation with ``conda`` on Windows
+--------------------------------------
+
+You can install and run ``unstructured`` on Windows with ``conda``, but the process
+involves a few extra steps. This section covers will help you get up and running.
```
Comment (qued) [inline docs/source/installing.rst:14]:
```suggestion
involves a few extra steps. This section will help you get up and running.
```

File: docs/source/installing.rst [hunk 2]
```diff
@@ -6,13 +6,85 @@ root directory. Developers can run ``make install-local`` to install the dev and
...
+* Run ``conda activate unstructured`` to activate the virtualenvironment.
+* Run ``pip install unstructured`` to install the ``unstructured`` library.
+
+===============================================
+Setting up ``unstructured`` for local inference
+===============================================
+
+If you need to run model inferences locally, there are a few additional steps you need to
+take. The main challenge is installing ``detectron2`` for PDF layout parsing. ``detectron2``
+does not officially support Windows, but it is possible to get it install on Windows.
```
Comment (qued) [inline docs/source/installing.rst:28]:
```suggestion
does not officially support Windows, but it is possible to get it to install on Windows.
```

File: docs/source/installing.rst [hunk 3]
```diff
@@ -6,13 +6,85 @@ root directory. Developers can run ``make install-local`` to install the dev and
...
+The installation instructions are based on the instructions LayoutParser provides
+`here <https://layout-parser.github.io/tutorials/installation#for-windows-users>`_.
+
+* Run ``pip install pycocotools-windows`` to install a Windows compatible version of ``pycocotools``. Alternatively, you can run ``pip3 install "git+https://github.com/philferriere/cocoapi.git#egg=pycocotools&subdirectory=PythonAPI"`` as outlined in `this GitHub issue <https://github.com/cocodataset/cocoapi/issues/169#issuecomment-462528628>`_.
+* Run ``git clone https://github.com/ivanpp/detectron2.git && cd detectron2 && pip install -e .`` to install a Windows compatible version of the ``detectron2`` library.
```
Comment (qued) [inline docs/source/installing.rst:33]: Stringing together commands via `&&` works in *sh and seems to work in a windows command prompt, but doesn't work in PowerShell. We might want to specify the terminal type.

File: docs/source/installing.rst [hunk 4]
```diff
@@ -6,13 +6,85 @@ root directory. Developers can run ``make install-local`` to install the dev and
...
+* Install the a Windows compatible version of ``iopath`` using the instructions outlined in `this GitHub issue <https://github.com/Layout-Parser/layout-parser/issues/15#issuecomment-819546751>`_. First, run ``git clone https://github.com/facebookresearch/iopath --single-branch --branch v0.1.8``. Then on line 753 in ``iopath/iopath/common/file_io.py`` change ``filename = path.split("/")[-1]`` to ``filename = parsed_url.path.split("/")[-1]``. After that, navigate to the ``iopath`` directory and run ``pip install -e .``.
+* Run ``pip install unstructured[local-inference]``. This will install the ``unstructured_inference`` dependency.
+
+At this point, you can verify the installation by running the following from the root directory of the ``unstructured-inference`` `repo <https://github.com/Unstructured-IO/unstructured-inference>`_:
```
Comment (qued) [inline docs/source/installing.rst:37]: Is there a way we could verify the install from this repo?

Comment (qued) [reply docs/source/installing.rst:37]: What works for me is:
```python
from unstructured.partition.pdf import partition_pdf

partition_pdf("example-docs/layout-parser-paper-fast.pdf", url=None)
```

File: docs/source/installing.rst [hunk 5]
```diff
@@ -6,13 +6,85 @@ root directory. Developers can run ``make install-local`` to install the dev and
...
+* Set the environment variable ``KMP_DUPLICATE_LIB_OK`` to ``"TRUE"``. This prevents the ``libiomp5md.dll`` linking issue described `in this issue on GitHub <https://github.com/PaddlePaddle/PaddleOCR/issues/4613>`_.
+
+
+At this point, you can verify the installation using the following commands. Choose a
+``.jpg`` image that contains text.
+
+.. code:: python
+
+	import numpy as np
+	from PIL import Image
```
Comment (qued) [inline docs/source/installing.rst:68]:
```suggestion
	from PIL import Image
        from paddleocr import PaddleOCR
```

File: docs/source/installing.rst (hunk 3 above)
Comment (MthwRobinson) [reply docs/source/installing.rst]: Added a comment on the terminal type

File: docs/source/installing.rst (hunk 4 above)
Comment (qued) [reply docs/source/installing.rst:37]: Don't forget to change line 37 (need to run this from root directory of `unstructured` not `unstructured-inference`).

**Review by qued — APPROVED** [review]
LGTM!

**Pull request merged**